_docs/_latest/develop/dev-setup.md

```diff
@@ -85,7 +88,7 @@ $ brew install openssl
 
 ### Installing mysql_config
 
-The MySql-python library requires the `mysql_config` utility to be present in your system.
+The MySQL-python library requires the `mysql_config` utility to be present in your system.
```

Comment:
This should come if the user follows the same steps in .travis.yml

Reply:
I am not sure what this means.  Can you explain it differently?

Reply:
Meaning, they should get mysql_config if they follow the same setup instructions that travis takes. My issue is that we don't duplicate setup steps and create a divergence problem (what we had before).